### ISSUE TITLE
[ISSUE] Accessibility for cardIconView in VGSCardTextField

### ISSUE THREAD

**Issue by AamirMir99** [verygoodsecurity/vgs-collect-ios#432]
Description:

Currently, the cardIconView inside VGSCardTextField does not allow setting an accessibilityLabel. As a result, VoiceOver cannot announce the card brand (e.g., “Visa”, “Mastercard”).

To Reproduce
	1.	Create a VGSCardTextField.
	2.	Enter a card number to trigger a card brand icon (e.g., Visa).
	3.	Enable VoiceOver on iOS.
	4.	Navigate to the cardIconView.
	5.	VoiceOver announces nothing .

Possible Solution

Inside the SDK, the `updateCardImage()` implementation already sets the image for cardIconView based on the card brand. This function could also be used to assign an accessibility label: 
    
`cardIconView.accessibilityLabel = state.cardBrand.stringValue`
  `icardIconView.accessibilityLabel.isAccessibilityElement = true`
